recipe/meta.yaml

```
@@ -1,7 +1,10 @@
 # Set Version and Build-Number
-{% set version = "2.6.3" %}
+{% set version = "2.7.0rc0" %} # in source specified as 2.7.0pre0
```

```suggestion
{% set version = "2.7.0rc0" %}
```
Maybe this could fix the issue

---

The last time it failed this was not there, but I can give it i a try. I wonder if the reason could that in the source code we use the identifier "2.7.0pre0" while in pypi it was translated to "2.7.0rc0".